test/unit/controllers/registration/NameDobControllerSpec.scala

```diff
@@ -158,7 +158,8 @@ class NameDobControllerSpec extends ControllerSpec with BeforeAndAfterEach with
     }
 
     "not be in the future" in {
-      val tomorrow = LocalDate.now().plusDays(1)
+      val tomorrow   = LocalDate.now().plusDays(1)
+      val FutureDate = "You must specify a date that is not in the future"
```

Comment:
what is this new val used for?

Reply:
This is moved from the ControllerSpec. It was defined there but used only here or here and one more place